ref(sentry apps): Reorganize main integration platform page (#5011)
getsentry/sentry-docs: src/docs/product/integrations/integration-platform/ui-components/index.mdx

---
title: UI Components
sidebar_order: 4
redirect_from:
  - /workflow/integrations/integration-platform/ui-components/

> description: "Learn more about Sentry's Integration Platform User Interface (UI) components and how they provide developers with the ability to define UI components to render within specific parts of Sentry's UI, through a JSON-based schema."

Reviewer: ```suggestion
description: "Learn more about Sentry's Integration Platform User Interface (UI) components and how they provide you with the ability to define UI components to render within sentry.io, through a JSON-based schema."
```
